client/src/app/auth.guard.ts

```diff
@@ -2,28 +2,40 @@ import { jsonDecoder } from 'src/app/utils/json.util';
 import { Injectable } from '@angular/core';
 import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot,Router } from '@angular/router';
 import { Observable } from 'rxjs';
-import { ServicesService } from './services.service';
+
+
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthGuard implements CanActivate {
-  constructor(private router:Router,private _service:ServicesService
+  constructor(private router:Router
     ){}
   canActivate(
     next: ActivatedRouteSnapshot,
     state: RouterStateSnapshot):  boolean{
      // this will be passed from the route config
     // on the data property
+    const token=localStorage.getItem('Authorization');
+    
     const expectedRole = next.data.expectedRole;
-    const decodeToken = jsonDecoder();
+    console.log(token);
+   
+    // decode the token to get its payload
+   
```

Comment:
remove console.log, blank lines and commented code

Reply:
done